WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl

```diff
@@ -54,47 +54,52 @@ void ctrlPPCheck()
   mapping map; // mapping with checked data
 
   // get current script path;
-  if ( isFunctionDefined("seGetFileName") )
-   path = seGetFileName();
+  if (isFunctionDefined("seGetFileName"))
+    path = seGetFileName();
 
   // store path;
   map["path"] = path;
 
   bool tmpFileUsed = FALSE;
-  if ( path == "" ) // panel or version < 3.16
+
+  if (path == "")   // panel or version < 3.16
```

Comment:
Consider refactoring: better check path.isEmpty()

Reply:
Do we want it?, It will never works with WinCC OA < 3.18

Reply:
Not necessary - was only a comment. But you are right for support-versions.